lib/ts/recipe/emailverification/api/emailVerify.ts

```diff
@@ -67,7 +68,7 @@ export default async function emailVerify(recipeInstance: Recipe, req: Request,
                 }
             })
         );
-        let session = (req as SessionRequest).session;
+        let session = (req as SessionRequest).session as SessionClass;
```

Comment:
The point of doing as `SessionRequest` is that we don't have to add another as `SessionClass`. So please check your types...
If you are getting type of Session has `undefined | SessionClass`, then have an `undefined` check below and throw a GENERAL_ERROR in that case, with a message like "should never come here" or something